label_sync/labels.yaml

```diff
@@ -265,6 +265,41 @@ default:
       prowPlugin: size
       addedBy: prow
 
+    #############################################################################
+    # Prioritisation labels borrowed from Kubernetes
+    #############################################################################
+
+    - color: fef2c0
+      description: Lowest priority. Possibly useful, but not yet enough support to actually get it done. # These are mostly place-holders for potentially good ideas, so that they don't get completely forgotten, and can be referenced /deduped every time they come up.
+      name: priority/awaiting-more-evidence
+      target: both
+      prowPlugin: label
+      addedBy: anyone
+    - color: fbca04
+      description: Higher priority than priority/awaiting-more-evidence. # There appears to be general agreement that this would be good to have, but we may not have anyone available to work on it right now or in the immediate future. Community contributions would be most welcome in the mean time (although it might take a while to get them reviewed if reviewers are fully occupied with higher priority issues, for example immediately before a release).
+      name: priority/backlog
+      target: both
+      prowPlugin: label
+      addedBy: anyone
+    - color: e11d21
+      description: Highest priority. Must be actively worked on as someone's top priority right now. # Stuff is burning. If it's not being actively worked on, someone is expected to drop what they're doing immediately to work on it. Team leaders are responsible for making sure that all the issues, labeled with this priority, in their area are being actively worked on. Examples include user-visible bugs in core features, broken builds or tests and critical security issues.
+      name: priority/critical-urgent
+      target: both
+      prowPlugin: label
+      addedBy: anyone
+    - color: eb6420
+      description: Important over the long term, but may not be staffed and/or may need multiple releases to complete.
+      name: priority/important-longterm
+      target: both
+      prowPlugin: label
+      addedBy: anyone
+    - color: eb6420
+      description: Must be staffed and worked on either currently, or very soon, ideally in time for the next release.
+      name: priority/important-soon
+      target: both
+      prowPlugin: label
+      addedBy: anyone
```

Comment:
These seem to be what one would use the project board for which sorta captures these states already.

Reply:
It doesn't today.

https://github.com/orgs/knative/projects/40/views/1

For example, we have 10 items ready to work but they don't have the priority labels so everyone doesn't know which one is important, work that we have deffered

Reply:
![image](https://user-images.githubusercontent.com/84356882/169846518-dad05573-0d3e-4bc7-9041-3c34036d2803.png)
Notice you can even title the priority field as you like

Reply:
I didn't see that column. Cool